example-client/README.md

```diff
@@ -3,11 +3,219 @@ Example CoAP client
 
 This is an example coap client application, that can be use with any LwM2M Server.
 
+# Running
+
 Usage
 -----
 
     ./run.sh <registration-url> [<keystore-file>]
 
+note: parameters are parsed like a [www URL query string](https://en.wikipedia.org/wiki/Query_string) where ? and & are delimeters for the endpoint name and account ID
+
 *For example:*
-        
-    ./run.sh 'coap://localhost:5683/rd?ep=device007&lt=3600&b=U'    
```

Comment:
This is still valid example, maybe we could keep it

Reply:
Again, that is defined in OMA LwM2M.

Reply:
Sadly that's not mentioned in the README as it stands before this PR so users have no way to know that information.

Reply:
True, that's clearly missing.